android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -429,6 +429,11 @@ public void setMessagingModuleName(WebView view, String moduleName) {
 
   @ReactProp(name = "incognito")
   public void setIncognito(WebView view, boolean enabled) {
+    // Don't do anything when incognito is disabled
+    if (!enabled) {
+      return;
+    }
```

Comment:
I think, for clarity, since we know that `enabled` is now `true` from this point onward, let's hardcode the `true | false` values of the following method calls rather than doing `!enabled`.

```javascript
view.getSettings().setAppCacheEnabled(false);

view.clearCache(true);

view.getSettings().setSavePassword(false);
view.getSettings().setSaveFormData(false);
```

Reply:
Hey nice catch, I have updated the flags. Can you please verify they are set correct?

Reply:
@jasonkellydk the code sniper @jamonholmgren should be deleted as well.

Reply:
Yup, looks good! Thanks @jasonkellydk !